components/bootloader/src/main/esp32.bootloader.ld

```diff
@@ -21,7 +21,7 @@ MEMORY
 }
 
 /*  Default entry point:  */
-ENTRY(call_start_cpu0);
+ENTRY(boot_loader_entry);
```

Comment:
fair enough, done.
(my goal was to make boot loader entrypoint name distinct from the app entry point name)